lib/compat/wordpress-6.1/class-wp-theme-json-resolver-6-1.php

```diff
@@ -80,8 +80,7 @@ public static function get_user_data() {
 
 		$config   = array();
 		$user_cpt = static::get_user_data_from_wp_global_styles( wp_get_theme() );
-
-		if ( array_key_exists( 'post_content', $user_cpt ) ) {
+		if ( is_array( $user_cpt ) && array_key_exists( 'post_content', $user_cpt ) ) {
```

Comment:
```suggestion
		if ( isset( $user_cpt['post_content'] ) ) {
```

Why not use `isset()` for the check here? It won't cause an error if the `$user_cpt` isn't an array. See: https://3v4l.org/4MYYN.

Reply:
When would this not be an array?